apps/wing/src/commands/run.ts

```diff
@@ -2,21 +2,53 @@ import { readdirSync, existsSync } from "fs";
 import { debug } from "debug";
 import { resolve } from "path";
 import open from "open";
+import { createConsoleApp } from "@wingconsole/app";
+import { parseNumericString } from "../util";
 
-export async function run(simfile?: string) {
-  if (!simfile) {
+/**
+ * Run options for the `run` command.
+ * This is passed from Commander to the `run` function.
```

Comment:
Maybe something like

```suggestion
 * Customizes the behavior of the created Wing Console
```